tests/fixtures/git/github.com/demo/demo/demo.egg-info/PKG-INFO

```diff
@@ -6,9 +6,5 @@ Home-page: https://github.com/demo/demo
 Author: Sébastien Eustace
 Author-email: sebastien@eustace.io
 License: MIT
-Platform: UNKNOWN
 Provides-Extra: foo
 Provides-Extra: bar
-
-UNKNOWN
-
```

Comment:
These are changes made by flaky test and shouldn't be part of your changes. Please revert that file.

Reply:
Okay i will revert the file